examples/v2/project_creation/project.py

```diff
@@ -50,7 +50,8 @@ def GenerateConfig(context):
       'properties': {
           'project': project_id,
           'billing': billing_name,
-          'apis': context.properties['apis']
+          'apis': context.properties['apis'],
+          'concurrent_api_activation': context.properties['concurrent_api_activation']
```

Comment:
Blergh. Sorry, Jai. This line is too long. Linter says it needs to be <80 and it's at 86. How strict do we want to be with the linting on this repo?